dds/DCPS/DomainParticipantFactoryImpl.cpp

```diff
@@ -6,7 +6,7 @@
  */
 
 #include "DCPS/DdsDcps_pch.h" //Only the _pch include should start with DCPS/
-
+#include "dds/DCPS/transport/framework/TransportRegistry.h"
```

Comment:
```suggestion
#include "transport/framework/TransportRegistry.h"
```

Also please more this below, as it says in the guidelines.